templates/accounts.conf.j2

```diff
@@ -20,6 +20,11 @@ accounts: {
         }
 {%     endfor %}
       ]
+
+{% if nats_jetstream_enabled|bool %}
+    jetstream: enabled
```

Comment:
Seems like this should be configured [at the top level first](https://docs.nats.io/running-a-nats-service/configuration/resource_management#enabling-jetstream-for-a-nats-server), did you check such config?

Reply:
@oukooveu it may be important to allow jetstream to limit resource use per account. However a cleaner approach would be if the account creation can be done explicitly maybe with the introduction of a new parameter `nats_accounts`.  I have gotten rid of this configuration here.

Reply:
> it may be important to allow jetstream to limit resource use per account

agree, but such possibility was not provided and are you sure this will work without global settings (see link provided above)?

Reply:
I have removed this account configuration

Reply:
But there is no such configuration anywhere, are you sure JetStream will work without ` jetstream: enabled`?